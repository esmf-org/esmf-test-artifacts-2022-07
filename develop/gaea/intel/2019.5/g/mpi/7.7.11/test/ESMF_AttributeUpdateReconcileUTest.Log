build time -- 2022-06-28 00:55:50
20220628 014323.752 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220628 014323.754 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220628 014323.754 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220628 014323.754 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220628 014323.754 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220628 014323.754 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220628 014323.754 INFO             PET0 Running with ESMF Version   : v8.4.0b01-2-gf0d4ab211d
20220628 014323.754 INFO             PET0 ESMF library build date/time: "Jun 28 2022" "00:53:34"
20220628 014323.754 INFO             PET0 ESMF library build location : /lustre/f2/dev/ncep/Mark.Potts/intel_2019.5_mpi_g_develop
20220628 014323.754 INFO             PET0 ESMF_COMM                   : mpi
20220628 014323.758 INFO             PET0 ESMF_MOAB                   : enabled
20220628 014323.758 INFO             PET0 ESMF_LAPACK                 : enabled
20220628 014323.758 INFO             PET0 ESMF_NETCDF                 : enabled
20220628 014323.758 INFO             PET0 ESMF_PNETCDF                : disabled
20220628 014323.758 INFO             PET0 ESMF_PIO                    : enabled
20220628 014323.758 INFO             PET0 ESMF_YAMLCPP                : enabled
20220628 014323.760 INFO             PET0  Beginning Test, file ESMF_AttributeUpdateReconcileUTest.F90, line 383
20220628 014323.761 INFO             PET0  NUMBER_OF_PROCESSORS           2
20220628 014323.799 INFO             PET0  PASS  PET            0 : Getting an updated Attribute value from a Component test: value = doodle, ESMF_AttributeUpdateReconcileUTest.F90, line 476
20220628 014323.799 INFO             PET0  PASS  PET            0 : Getting an updated Attribute value from a Component test: value = doodle, ESMF_AttributeUpdateReconcileUTest.F90, line 491
20220628 014323.801 INFO             PET0  Ending Test, file ESMF_AttributeUpdateReconcileUTest.F90, line 515
20220628 014323.801 INFO             PET0 Finalizing ESMF
20220628 014323.752 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220628 014323.754 INFO             PET1 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220628 014323.754 INFO             PET1 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220628 014323.754 INFO             PET1 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220628 014323.754 INFO             PET1 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220628 014323.754 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220628 014323.754 INFO             PET1 Running with ESMF Version   : v8.4.0b01-2-gf0d4ab211d
20220628 014323.754 INFO             PET1 ESMF library build date/time: "Jun 28 2022" "00:53:34"
20220628 014323.754 INFO             PET1 ESMF library build location : /lustre/f2/dev/ncep/Mark.Potts/intel_2019.5_mpi_g_develop
20220628 014323.754 INFO             PET1 ESMF_COMM                   : mpi
20220628 014323.758 INFO             PET1 ESMF_MOAB                   : enabled
20220628 014323.758 INFO             PET1 ESMF_LAPACK                 : enabled
20220628 014323.758 INFO             PET1 ESMF_NETCDF                 : enabled
20220628 014323.758 INFO             PET1 ESMF_PNETCDF                : disabled
20220628 014323.758 INFO             PET1 ESMF_PIO                    : enabled
20220628 014323.758 INFO             PET1 ESMF_YAMLCPP                : enabled
20220628 014323.760 INFO             PET1  Beginning Test, file ESMF_AttributeUpdateReconcileUTest.F90, line 383
20220628 014323.761 INFO             PET1  NUMBER_OF_PROCESSORS           2
20220628 014323.799 INFO             PET1  PASS  PET            1 : Getting an updated Attribute value from a Component test: value = doodle, ESMF_AttributeUpdateReconcileUTest.F90, line 476
20220628 014323.799 INFO             PET1  PASS  PET            1 : Getting an updated Attribute value from a Component test: value = doodle, ESMF_AttributeUpdateReconcileUTest.F90, line 491
20220628 014323.801 INFO             PET1  Ending Test, file ESMF_AttributeUpdateReconcileUTest.F90, line 515
20220628 014323.801 INFO             PET1 Finalizing ESMF
