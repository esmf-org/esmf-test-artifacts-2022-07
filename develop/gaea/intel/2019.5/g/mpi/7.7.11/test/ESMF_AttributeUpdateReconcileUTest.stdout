build time -- 2022-07-06 01:17:54
  Beginning Test, file ESMF_AttributeUpdateReconcileUTest.F90, line 383
  Beginning Test, file ESMF_AttributeUpdateReconcileUTest.F90, line 383
  NUMBER_OF_PROCESSORS           2
  NUMBER_OF_PROCESSORS           2
  PASS  PET            0 : Getting an updated Attribute value from a Component t
 est: value = doodle, ESMF_AttributeUpdateReconcileUTest.F90, line 476
  PASS  PET            1 : Getting an updated Attribute value from a Component t
 est: value = doodle, ESMF_AttributeUpdateReconcileUTest.F90, line 476
  PASS  PET            0 : Getting an updated Attribute value from a Component t
 est: value = doodle, ESMF_AttributeUpdateReconcileUTest.F90, line 491
  PASS  PET            1 : Getting an updated Attribute value from a Component t
 est: value = doodle, ESMF_AttributeUpdateReconcileUTest.F90, line 491
  Ending Test, file ESMF_AttributeUpdateReconcileUTest.F90, line 515
  Ending Test, file ESMF_AttributeUpdateReconcileUTest.F90, line 515
  PET           0  Test Elapsed Time    7.90299999999999       msec.
  PET           1  Test Elapsed Time    9.83800000000001       msec.
